data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
I am not sure what you mean.  This is fixing the result for Firefox 20..27, which was wrong.
No browsers were removed, but I did consolidate identical columns of the table.